### ISSUE TITLE
Typo in the description of SupLoss

### ISSUE THREAD

**Issue by bpascal-fr** [deepinv/deepinv#461]
I feel like the normalization by the total number of elements is missing in the description of SupLoss.

Indeed, SupLoss is, by default, calling `torch.nn.MSELoss` with default parameters and thus with `reduction = 'mean'`.

https://github.com/deepinv/deepinv/blob/dca7d9419eef7bd4bf23102719a47adb78e036d4/deepinv/loss/sup.py#L12-L18

**Comment by tachella** [comment]
thanks for spotting this, we will fix it asap!
